sources/BringgTrackingClient.m

```diff
@@ -149,22 +149,89 @@ - (nullable GGCustomer *)signedInCustomer{
 
 
 - (void)startWatchingOrderWithUUID:(NSString *_Nonnull)uuid
-                        sharedUUID:(NSString *_Nullable)shareduuid
```

Comment:
Again, change all shareduuid to shareUUID or sharedUUID (will stop commenting regarding this)